pkgs/pinned.nix

```diff
@@ -1,13 +1,16 @@
 let
   nixpkgsPinned = import ./nixpkgs-pinned.nix;
-  unstable = import nixpkgsPinned.nixpkgs-unstable { config = {}; overlays = []; };
-  nixBitcoinPkgsUnstable = import ./. { pkgs = unstable; };
+  nixpkgs-unstable = import nixpkgsPinned.nixpkgs-unstable { config = {}; overlays = []; };
+  nixpkgs-stable = import nixpkgsPinned.nixpkgs { config = {}; overlays = []; };
 in
-{
+rec {
   inherit (unstable)
     bitcoin
     bitcoind
     clightning
     lnd;
-  inherit (nixBitcoinPkgsUnstable) electrs;
+  inherit (stable) electrs;
```

Comment:
This must be inherited the unstable nix-bitcoin packages, as before.

Reply:
oh right this is a typo, thanks